components/AnchorNavigation/src/index.scss

```diff
@@ -0,0 +1,57 @@
+.denhaag-anchor-navigation {
+  border-inline-start: var(--denhaag-anchor-navigation-border-inline-start);
+}
+
+.denhaag-anchor-navigation .denhaag-anchor-navigation__list {
+  list-style: var(--denhaag-anchor-navigation-list-list-style);
+  padding-inline-start: var(--denhaag-anchor-navigation-list-padding-inline-start);
+}
+
+.denhaag-anchor-navigation .denhaag-anchor-navigation__link {
+  font-family: var(--denhaag-anchor-navigation-link-font-family);
+  font-size: var(--denhaag-anchor-navigation-link-font-size);
+  line-height: var(--denhaag-anchor-navigation-link-line-height);
+  text-decoration: var(--denhaag-anchor-navigation-link-text-decoration);
+}
+
+.denhaag-anchor-navigation .denhaag-anchor-navigation__link,
```

Comment:
`.denhaag-anchor-navigation__link,
.denhaag-anchor-navigation__link:visited {` should be enough